frontend/src/metabase/common/hooks/use-has-metrics.tsx

```diff
@@ -0,0 +1,18 @@
+import type { SearchRequest } from "metabase-types/api";
+
+import { useFetchMetrics } from "./use-fetch-metrics";
+
+/** NOTE: hasModels is undefined when the request is pending */
+export const useHasMetrics = (req: Partial<SearchRequest> = {}) => {
```

Comment:
:x: 
I don't think this one should live in `common`, it's very specific to the Browse Metrics page due to `model_ancestors` and `filter_items_in_personal_collection` params.

See [useAvailableData](https://github.com/metabase/metabase/blob/35c84771208aea7b67eabedf5d22786210f6dcc7/frontend/src/metabase/common/components/DataPicker/hooks/useAvailableData.ts#L8) - it was also tempting to make it a "global" solution then, but see - it would not work here.

Although it should be possible to align these functions a bit, maybe even reuse.
Maybe we should do it in a separate PR.